snap/snapcraft.yaml

```diff
@@ -140,7 +140,7 @@ parts:
     shell: bash
     shell-flags: ['-eux', '-o', 'pipefail']
     shell-command: |
-      export GOROOT="$SNAPDIR/parts/go/install"
+      export GOROOT=$("$SNAPCRAFT_STAGE/bin/go env GOROOT")
```

Comment:
This is my typo, unfortunately this needs to be :
```
export GOROOT=$("$SNAPCRAFT_STAGE/bin/go" env GOROOT)
```
with quotes around just the `$SNAPCRAFT_STAGE/bin/go`, not around the whole subshell.